packages/db/src/lib/granule.ts

```diff
@@ -267,7 +267,7 @@ export const getGranulesByApiPropertiesQuery = (
 export const getGranulesByGranuleId = async (
   knexOrTransaction: Knex | Knex.Transaction,
   granuleId: string
-): Promise<Array<PostgresGranuleRecord>> => {
```

Comment:
Good catch.    Nit: docstring should be updated as well, (or we could have the discussion about removing the jsdoc lint requirement from TS files given the redundancy)